### ISSUE TITLE
`/bin/sh: rimraf: command not found` when installing menubar

### ISSUE THREAD

**Issue by julienma** [maxogden/menubar#225]
[//]: # "Hey there and thank you for using Menubar's issue tracker!"
[//]: # "Please fill in these information to describe your issue."

### Description

I'm using https://github.com/electron-react-boilerplate/electron-react-boilerplate, which has [some script to build native deps](https://electron-react-boilerplate.js.org/docs/adding-dependencies#which-packagejson-file-to-use) when running `yarn`.

When I add latest menubar, I get an error:

```sh
❯ yarn add --dev menubar
yarn add v1.16.0
[1/4] 🔍  Resolving packages...
[2/4] 🚚  Fetching packages...
[3/4] 🔗  Linking dependencies...
[4/4] 🔨  Building fresh packages...
[1/2] ⠂ fsevents
error /Users/julien/p/node_modules/menubar: Command failed.
Exit code: 127
Command: yarn build
Arguments:
Directory: /Users/julien/p/node_modules/menubar
Output:
yarn run v1.16.0
$ rimraf lib/
/bin/sh: rimraf: command not found
error Command failed with exit code 127.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

### Steps to Reproduce the Problem

1. `git clone git@github.com:electron-react-boilerplate/electron-react-boilerplate.git`
2. `yarn` => OK
3. `yarn dev` => OK
4. `yarn add --dev menubar` => fail

### Expected Behavior

`yarn` should install menubar without error.

### Specifications

- Menubar version: 6.0.4
- Platform: macOS 10.14.5
- Electron version: 5.0.3

**Comment by amaurymartiny** [comment]
Fixed in [`menubar@6.0.5`](https://www.npmjs.com/package/menubar/v/6.0.5) 🚀

**Comment by julienma** [comment]
Thanks!
